docs/azure/azure.rst

```diff
@@ -128,8 +128,9 @@ Adding Vault Creds To functions
 
 Function Code and Queries
 =========================
-* For standard Hybrid Commited Spend queries use the default `azure_function <https://github.com/project-koku/koku-data-selector/blob/main/docs/azure/scripts/azure-function.txt>`_
-* For custom queries non HCS we need to edit line 53 in the above function code.
+* The default script has the option of Hybrid commited spend or RHEL subscription filtering `azure_function <https://github.com/project-koku/koku-data-selector/blob/main/docs/azure/scripts/azure-function-hcs.txt>`_
```

Comment:
I think the name here is off. Should just be `/scripts/azure-function.txt` i think.
```suggestion
* The default script has the option of Hybrid commited spend or RHEL subscription filtering `azure_function <https://github.com/project-koku/koku-data-selector/blob/main/docs/azure/scripts/azure-function.txt>`_
```